lib/ruby-next/commands/nextify.rb

```diff
@@ -96,12 +130,13 @@ def parse!(args)
       private
 
       def transpile(path, contents, version: min_version)
-        rewriters = Language.rewriters.select { |rw| rw.unsupported_version?(version) }
+        rewriters = specified_rewriters || Language.rewriters.select { |rw| rw.unsupported_version?(version) }
 
         context = Language::TransformContext.new
 
         new_contents =
-          if Gem::Version.new(version) >= Gem::Version.new("2.7.0") && !defined?(Unparser::Emitter::CaseMatch)
+          if specified_rewriters ||
```

Comment:
Rewriters could be used for `Language.transform` as well (this also "rewriting" but using a different approach), so, I think, we don't need to check for the specified rewriters here.

Reply:
@palkan 

Ah, I see, it has nothing to rewriters themselves, just a fallback for `rewrite` mode if ruby is `2.7+` and `Unparser` has no `CaseMatch` defined. (https://github.com/mbj/unparser/pull/142)

It confused me because I get the output below with different versions of ruby on my machine if using `ast` transpile mode:
```
$ bin/ruby-next nextify spec/cli/dummy/
No emitter for node: :defs_e
```

AFAIU it's related to endless method definition which should be supported by `Unparser`, but it fails for me for some reason.

Reply:
> endless method definition which should be supported by Unparser,

I don't think it's supported by Unparser, it's from the upcoming Ruby version.

> It confused me because I get the output below with different versions of ruby on my machine if using ast transpile mode

Right. If you haven't enabled EndlessMethod rewriters, it should fail. Our Ruby 2.7 hack is not good, it doesn't catch this. I have an idea how to fix this, will try and push to master soon.

Reply:
Added https://github.com/ruby-next/ruby-next/commit/1fa44897dedcaef0f2beebfe71265df0c92305dd.

Could you please rebase? Now everything should work without additional hacks at the CLI level.

Reply:
BTW unparser 0.4.8 was released that now has a dedicated exceptions for unknown node types. See https://github.com/mbj/unparser/pull/150

Reply:
Thanks @mbj!